### PR TITLE
Design updates 2

### PR DESCRIPTION
**Pull request by isaacwilliams**
A whole bunch of little visual design tweaks.

### CODE REVIEW

File: src/components/CollapsedOpeningTimes.scss [hunk 1]
```diff
@@ -2,16 +2,18 @@
...
+        right: 10px;
+		font-size: 16px;
     }
 
     ul {
         margin: 0;
-        margin-left: 25px;
+        margin-left: 32px;
         padding: 0;
+		padding-bottom: 10px;
```
Comment (danni) [inline src/components/CollapsedOpeningTimes.scss]: Indenting is weird.

File: src/components/Collapser.js [hunk 2]
```diff
@@ -69,6 +64,19 @@ class Collapser extends React.Component {
             </div>
         );
     }
+	
+	renderMessage(): ReactElement {
+		if ( this.state.collapsed ) {
```
Comment (danni) [inline src/components/Collapser.js]: Spacing is weird.

File: src/components/Contact.scss [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
-    padding-top: 7px;
-}
 
 .Contact {
     display: block;
     width: 100%;
-
+    padding-top: 6px;
+	
+	.kind{
```
Comment (danni) [inline src/components/Contact.scss]: Missing space.

File: src/components/Contact.scss [hunk 4]
```diff
@@ -33,15 +38,41 @@
...
+		
+		border: 1px solid $brand-pane-alt-dark;
+		text-align: center;
+		border-radius: 3px;
+		text-decoration: none;
+        font-size: 18px;
+		
+        .Icon {
+           	position: absolute;
+			display: inline-block;
```
Comment (danni) [inline src/components/Contact.scss]: Weird indenting.

File: src/components/Contact.scss [hunk 5]
```diff
@@ -33,15 +38,41 @@
...
             height: 40px;
 
             path {
-                fill: $brand-pane-alt-dark;
+                fill: $brand-text-dark;
             }
         }
     }
+	
+	a.ContactButton{
```
Comment (danni) [inline src/components/Contact.scss]: Specifically `a.ContactButton`? or just `.ContactButton` ?

File: src/components/CrisisLineItem.js [hunk 6]
```diff
@@ -38,7 +38,7 @@ class CrisisLineItem extends React.Component {
                             <Link
                                 to={`/service/${object.slug}`}
                             >
-                                {object.site.name} {object.name}
+                                {object.site.name}
```
Comment (danni) [inline src/components/CrisisLineItem.js:41]: This is problematic IMO.

File: src/components/Email.js [hunk 7]
```diff
@@ -18,11 +18,10 @@ export default class Email extends React.Component {
...
                     <icons.Email />
                     <div className="Contact-text">
-                        <span className="kind">
-                            {this.props.comment ? this.props.comment
-                            : "Email"}
-                        </span>
-                        {' '}
+				        <span className="kind">
+				            {this.props.comment ? this.props.comment
+				            : ""}
```
Comment (danni) [inline src/components/Email.js]: I think you need something here still, even if it's marked as Screenreader only.

File: src/components/NavBar.scss [hunk 8]
```diff
@@ -3,3 +3,19 @@
         padding: 0;
     }
 }
+.FeedbackLink{
+	background: $brand-bg-light;
+	
+	> div{
```
Comment (danni) [inline src/components/NavBar.scss]: `& > div` to be consistent.

File: src/components/ServicePane.js [hunk 9]
```diff
@@ -58,16 +58,16 @@ export default class ServicePane extends React.Component {
 
                     <CollapsedOpeningTimes object={object.open} />
                     <hr />
-
+
```
Comment (danni) [inline src/components/ServicePane.js]: Trailing space.

File: src/components/ServicePane.scss [hunk 10]
```diff
@@ -40,37 +40,63 @@ $padding: 15px;
...
+	.TransportTime > .location{
+		display: none;
+	}
 
     .List {
         .ListItem {
             font-size: 14px;
             font-weight: 400;
+			
+			> div{
```
Comment (danni) [inline src/components/ServicePane.scss]: `& > div` to be consistent.

File: src/components/TransportTime.js [hunk 11]
```diff
@@ -52,18 +52,7 @@ class TransportTime extends React.Component {
...
-                    {compact: this.props.compact}
-                )}
-            >
-                <icons.Phone className="ColoredIcon brand-text-dark" />
-                <span className="travel-time">
-                    Confidential location
-                </span>&nbsp;
-                {this.renderSuburb()}
-            </div>
+            <div />
```
Comment (danni) [inline src/components/TransportTime.js]: This whole method is now a no-op, it can be removed.

File: src/components/TransportTime.js [hunk 12]
```diff
@@ -71,29 +60,42 @@ class TransportTime extends React.Component {
...
+		            {this.renderDirections()}
+		        </div>
+			</div>
+	        
+            
         );
     }
-
+	
+    renderDivider(): ReactElement {
```
Comment (danni) [inline src/components/TransportTime.js]: Suspect flow will complain this should be `?ReactElement`

File: src/pages/ResultsPage.js [hunk 13]
```diff
@@ -107,7 +107,7 @@ class ResultsPage extends BaseCategoriesPage {
             throw new Error("Unexpected");
         }
     }
-
+
```
Comment (danni) [inline src/pages/ResultsPage.js]: Trailing space.

File: src/pages/ResultsPage.js [hunk 14]
```diff
@@ -162,6 +162,7 @@ class ResultsPage extends BaseCategoriesPage {
         return (
             <div className="ResultsPage">
                 <AppBar
+					backMessage="Categories"
```
Comment (danni) [inline src/pages/ResultsPage.js]: Weird indenting.

**Comment by danni** [comment]
This is going to fail the style checkers. You can run them with `./script/typecheck` or by adding `.githooks` to `.git/hooks`.

File: src/components/CrisisLineItem.js (hunk 6 above)
Comment (isaacwilliams) [inline src/components/CrisisLineItem.js:41]: Define problematic.

File: src/components/Email.js (hunk 7 above)
Comment (isaacwilliams) [inline src/components/Email.js]: How do I do that?

File: src/components/CrisisLineItem.js (hunk 6 above)
Comment (danni) [inline src/components/CrisisLineItem.js:41]: It tells us who is operating the service, but not what it is.

File: src/components/AppBar.js [hunk 15]
```diff
@@ -45,18 +45,29 @@ class AppBar extends React.Component {
     render(): ReactElement {
         return (
             <div className="AppBar">
-                <components.IconButton
-                    className="BackButton button-container"
-                    onTouchTap={this.props.onBackTouchTap}
-                >
-                    <icons.ChevronBack className="left" />
-                </components.IconButton>
+				{this.renderBackButton()}
```
Comment (danni) [inline src/components/AppBar.js]: Wrong indent?

File: src/components/ContactMethods.scss [hunk 16]
```diff
@@ -1,5 +1,7 @@
 .ContactMethods {
+	padding-bottom: 10px;
     .collapser-message {
-        line-height: 3em;
+		padding-top: 10px;
+		display: block;
```
Comment (danni) [inline src/components/ContactMethods.scss]: Weird indenting.

File: src/components/LogoWithShadow.scss [hunk 17]
```diff
@@ -25,4 +25,7 @@
             fill: $brand-text-dark;
         }
     }
+	.LogoShadowIcon {
```
Comment (danni) [inline src/components/LogoWithShadow.scss]: Blank line between declarations.

File: src/components/Phone.js [hunk 18]
```diff
@@ -17,14 +17,13 @@ class Phone extends React.Component {
     render(): ReactElement {
         return (
             <div className="Contact Phone">
-                <a href={this.href}>
-                    <icons.Phone />
+		        <span className="kind">
+		            {this.props.comment ? this.props.comment
+		            : titleize(this.props.kind)}
+		        </span>
```
Comment (danni) [inline src/components/Phone.js]: Weird indenting.

File: src/components/Phone.js [hunk 19]
```diff
@@ -17,14 +17,13 @@ class Phone extends React.Component {
...
+                <a href={this.href} className="ContactButton">
                     <div className="Contact-text">
-                        <span className="kind">
-                            {this.props.comment ? this.props.comment
-                            : titleize(this.props.kind)}
-                        </span>
-                        {' '}
+	                    <icons.Phone />
```
Comment (danni) [inline src/components/Phone.js]: Weird indenting.

File: src/components/ResultListItem.scss [hunk 20]
```diff
@@ -3,7 +3,9 @@
 
     font-size: 14px;
     font-weight: 400;
-
+	
+	border-bottom: 10px solid $list-item-hairline;
```
Comment (danni) [inline src/components/ResultListItem.scss]: Weird indenting.

File: src/components/ServicePane.js [hunk 21]
```diff
@@ -58,16 +58,16 @@ export default class ServicePane extends React.Component {
...
-
+					
                     <GoogleMapsLink
                         className="plain-text"
                         from="Current Location"
                         to={object.Location()}
                     >
                         <Address location={object.Location()} />
-                        <TransportTime location={object.Location()}/>
+						<TransportTime location={object.Location()}/>
```
Comment (danni) [inline src/components/ServicePane.js]: Weird indenting.

File: src/components/ServicePane.js [hunk 22]
```diff
@@ -58,16 +58,16 @@ export default class ServicePane extends React.Component {
...
                     </GoogleMapsLink>
-
+
```
Comment (danni) [inline src/components/ServicePane.js]: Trailing space.

File: src/components/ServicePane.js [hunk 23]
```diff
@@ -79,8 +79,8 @@ export default class ServicePane extends React.Component {
             </div>
         );
     }
-
-    renderServiceProvisions(): ReactElement {
+
```
Comment (danni) [inline src/components/ServicePane.js]: Trailign space.

File: src/components/ServicePane.js [hunk 24]
```diff
@@ -79,8 +79,8 @@ export default class ServicePane extends React.Component {
             </div>
         );
     }
-
-    renderServiceProvisions(): ReactElement {
+	
+	renderServiceProvisions(): ReactElement {
```
Comment (danni) [inline src/components/ServicePane.js]: Weird indenting.

File: src/components/ServicePane.scss [hunk 25]
```diff
@@ -40,37 +40,63 @@ $padding: 15px;
...
     .description {
         padding-bottom: 10px;
         color: $brand-text-mid;
         font-weight: 400;
+		font-size: 18px;
+		line-height: 140%;
     }
 
-    .OpeningTimes,
+    .OpeningTimes{
```
Comment (danni) [inline src/components/ServicePane.scss]: Missing space before `{`

File: src/components/ServicePane.scss [hunk 26]
```diff
@@ -40,37 +40,63 @@ $padding: 15px;
...
-            top: -6px;
+            top: -2px;
         }
     }
 
     .OpeningTimes,
     .Address,
-    .CollapsedPhones {
+    .CollapsedPhones,
+	.TransportTime {
```
Comment (danni) [inline src/components/ServicePane.scss]: Weird indenting

File: src/components/ServicePane.scss [hunk 27]
```diff
@@ -40,37 +40,63 @@ $padding: 15px;
...
         padding-top: 10px;
         padding-bottom: 10px;
+		padding-left: 32px;
+		font-size: 16px;
     }
+	
+	.TransportTime > .location{
```
Comment (danni) [inline src/components/ServicePane.scss]: Missing space before `{`

File: src/components/ServicePane.scss [hunk 28]
```diff
@@ -40,37 +40,63 @@ $padding: 15px;
...
 
     .List {
         .ListItem {
             font-size: 14px;
             font-weight: 400;
+			
+			> div{
+				& > .primaryText{
+					font-weight: 500;
+				}
```
Comment (danni) [inline src/components/ServicePane.scss]: Missing line between declarations.

File: src/components/ServicePane.scss [hunk 29]
```diff
@@ -40,37 +40,63 @@ $padding: 15px;
...
+		            white-space: normal;
+		            overflow: auto;
+		        }
+			}
+			
         }
     }
 
     .siblings {
+		padding-top: 5px;
```
Comment (danni) [inline src/components/ServicePane.scss]: Weird indenting. New line between this and child declarations.

File: src/pages/HomePage.scss [hunk 30]
```diff
@@ -42,11 +50,11 @@
     .search {
         position: relative;
         width: 100%;
-        padding-bottom: 20px;
+        padding-bottom: 15px;
 
         input {
             display: block;
-            width: 90%;
+			width: 90%;
```
Comment (danni) [inline src/pages/HomePage.scss]: Weird indenting.

File: src/pages/ResultsListPage.scss [hunk 31]
```diff
@@ -12,11 +12,23 @@
...
+		width: 40px;
+		height: 40px;
+	}
 
     .change-personalisation {
-        color: $white;
-        font-size: 130%;
-        font-weight: 600;
+		font-weight: 400;
+		color: $white;
```
Comment (danni) [inline src/pages/ResultsListPage.scss]: Weird indenting.

**Comment by danni** [comment]
Handful of tests are failing because the content has changed. You'll find them in the `.feature` files :)

File: src/components/CrisisLineItem.js (hunk 6 above)
Comment (isaacwilliams) [inline src/components/CrisisLineItem.js:41]: In theory, yes. But in practice you get giant blocks of text with lots of repeated data. Really hard to read and the user would be more confused than informed.

**Comment by isaacwilliams** [comment]
Updated to pass all the tests that were my fault. There's still a couple failing which I think aren't my jurisdiction.

**Comment by DanielHeath** [comment]
I'll have a look.

**Comment by DanielHeath** [comment]
screen shot 2015-11-26 at 12 21 57 pm

A few things are looking odd at http://localhost:8000/service/107541-law-society-of-new-south-wales on this branch.

Should the email / website be centred if the phone is centred?

**Comment by DanielHeath** [comment]
Isaac: On the personalisation summary screen there's no longer an "Okay" button to take you back to results - the back button does it (confirming that's the design intent).

**Comment by isaacwilliams** [comment]
Daniel:
- I agree the phone number looks weird on a wide screen. I'm just working on some tweaks to make that look a bit better.
- That is intent, correct.

File: src/components/CrisisLineItem.js (hunk 6 above)
Comment (danni) [inline src/components/CrisisLineItem.js:41]: The problem is you just get a name but it tells you nothing about what you'll get when you call. I.e. is it an IAP, or an infoline, etc. What would be better maybe is to apply some substitutions, like `/IAP/` to "assessment line" etc.

Comment (danni) [inline src/components/CrisisLineItem.js:41]: Although in this specific case I think we removed the IAPs from being hotlines.

**Pull request merged**